packages/runtime-core/src/vnode.ts

```diff
@@ -214,6 +215,11 @@ export function createVNode(
     type = Comment
   }
 
+  // class component normalization.
+  if (isFunction(type) && '__vueClassComponentOptions' in type) {
```

Comment:
Can we use a shorter key to save a few bytes? Maybe `__vccOpts`? (I assume this key will be fully internal and never exposed to the user)

Reply:
Sure, renamed.